core/cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/consent/ConsentProperties.java

```diff
@@ -18,6 +18,13 @@
  */
 public class ConsentProperties implements Serializable {
     private static final long serialVersionUID = 5201308051524438384L;
+    
+    /**
+     * CAS server login URL to use. 
+     * Protect the consent overview endpoints via CAS itself.
+     */
+    private String loginUrl;
```

Comment:
This needs to go away. You already have the login url under `CasServerProperties`:
https://github.com/apereo/cas/blob/master/core/cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/core/CasServerProperties.java#L134